### PR TITLE
added the appveyor build for Windows

### PR DESCRIPTION
**Pull request by pratik0316**
In this PR I have added the appveyor builds for Ubuntu 18.04LTS.
The link to the [logs](https://drive.google.com/drive/folders/1ajWivCoL_lFFwryqTodu5NhAU40EuVRK?usp=sharing)

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+image: Ubuntu
```
Comment (steven-esser) [inline appveyor.yml]: We want windows for appveyor.

**Review by steven-esser — CHANGES_REQUESTED** [review]

File: appveyor.yml (hunk 1 above)
Comment (pratik0316) [reply appveyor.yml]: @majurg sure I will be adding the windows build too.

**Comment by pratik0316** [comment]
@majurg , I added the windows build to Appveyor [logs](https://drive.google.com/file/d/13OZBBoBZ2f0sRKPD9Cw-dTtLg6WqpBq-/view?usp=sharing)

**Comment by pratik0316** [comment]
@majurg , Please review it again I have made the required changes

**Comment by steven-esser** [comment]
We have made the decision to move to azure pipelines for ci, as it supports all OSes. Thanks. Closing this.

**Pull request closed without merging**